lib/facter/framework/logging/logger.rb

```diff
@@ -3,10 +3,11 @@
 require 'logger'
 
 module Facter
-  RED = 31
-  GREEN = 32
-  YELLOW = 33
-  CYAN = 36
+  RED     = "\e[31m"
+  GREEN   = "\e[32m"
+  YELLOW  = "\e[33m"
```

Comment:
\e[0;{COLOR_CODE}m https://www.cyberciti.biz/faq/bash-shell-change-the-color-of-my-shell-prompt-under-linux-or-unix/

Reply:
The test is quite bad, it doesn't actually test that the output has color. It tests that the output text is not formatted.
I'd rather fix the test.